Source/rFactor2SharedMemoryMap.cpp

```diff
@@ -962,7 +962,7 @@ void SharedMemoryPlugin::ReadHWControl()
     return;
 
   // Control the rate of reads.
-  auto const needsBoost = ++mHWControlRequestBoostCounter < 5;  // 100ms boost.
+  auto const needsBoost = ++mHWControlRequestBoostCounter < 25;  // 500ms boost.

```

Comment:
thanks - please update readme.md and top level .cpp comment to mention this delay